### PR TITLE
Update 04.use-cases-b.md

### PR DESCRIPTION
**Pull request by emmajhudgins**
Added a few paragraphs on manuscript writing. @Aariq please let me know if you have things to add to the Quarto section largely pulled from your Issue #145

### CODE REVIEW

File: content/04.use-cases-b.md [hunk 1]
```diff
@@ -78,16 +78,36 @@ Periodic code review can also help advisors to identify errors early in the proc
...
-We used this platform early on in the process of writing this manuscript to generate an outline.
+<!--*contributors to this section:* Emma J. Hudgins-->
 
-Later, we wrote the manuscript by Manubot, which is one of the important collaborative manuscript platform uses Markdown for writing and GitHub for storing and tracking changes over time [@https://doi.org/10.1371/journal.pcbi.1007128]
+Beyond supporting collaboration at the level of code, GitHub can even be used for collaborative writing of manuscripts. 
+Writing a manuscript in GitHub and storing it with the data and code associated with a repository allows for all files associated with a research product to be found in one place.
+Further, authors can make use of the Discussions tab to suggest relevant papers to be cited, and can raise issues during the writing process that can be assigned to collaborators.
+
+While GitHub is not considered as user-friendly for manuscript development as conventional word processors (Ram 2013), it has been substantially improved with recent tools.
+An important collaborative tool is [HackMD](https://hackmd.io), allows real-time collaboration like Google Docs for individual Markdown documents.
```
Comment (Aariq) [inline content/04.use-cases-b.md:88]:
```suggestion
An important collaborative tool is [HackMD](https://hackmd.io), which allows real-time collaboration like Google Docs for individual Markdown documents. 
```

Have we explained what Markdown is at this point in the manuscript?  If not, I think the first point to make is that although GitHub doesn't work well with Word, it _does_ work well with Markdown which can be rendered to Word, PDF, and HTML.

File: content/04.use-cases-b.md [hunk 2]
```diff
@@ -78,16 +78,36 @@ Periodic code review can also help advisors to identify errors early in the proc
...
+Further, authors can make use of the Discussions tab to suggest relevant papers to be cited, and can raise issues during the writing process that can be assigned to collaborators.
+
+While GitHub is not considered as user-friendly for manuscript development as conventional word processors (Ram 2013), it has been substantially improved with recent tools.
+An important collaborative tool is [HackMD](https://hackmd.io), allows real-time collaboration like Google Docs for individual Markdown documents. 
+At any time, changes to documents can be committed to an associated GitHub repository.
+We used this platform early in the process of writing this manuscript to generate an outline.
+
+Later, we wrote this manuscript using Manubot, a collaborative manuscript platform that uses Markdown for writing and GitHub for storing and tracking changes to a manuscript over time [@https://doi.org/10.1371/journal.pcbi.1007128]. 
+Manubot uses a GitHub Actions-based typesetting system to compiles individual Markdown documents stored in a GitHub repository into a single LaTeX document, which can be displayed in HTML or PDF formats.
```
Comment (Aariq) [inline content/04.use-cases-b.md:93]:
```suggestion
Manubot uses a GitHub Actions-based typesetting system to compile individual Markdown documents stored in a GitHub repository into a single LaTeX document, which can be displayed in Word, HTML, or PDF formats.
```
Manubot has an option to create word documents---we just haven't used it I think.

**Review by Aariq — COMMENTED** [review]
I think I like the order of things, but I don't know if manubot and quarto really need their own paragraphs.  Here's an outline that I'm thinking might focus on the big picture a little and bring manubot and quarto in more as examples

1) Benefits: 
- version control to track changes in manuscript.  
- Pull requests as a sane way of merging contributions from multiple co-authors. 
- Add-ons like HackMD that allow real-time collaborative editing
- Keep code and text in one place allows for fully reproducible manuscripts

2) Challenge: familiar formats like Word docs don't play nicely with GitHub because it wants plain text.

3) Solutions: 
- Markdown is a simple, easy to learn markup language to format plain text into Word, PDF, or HTML using Pandoc.  
- Many tools exist for extending Markdown and Pandoc to add formatting features necessary for scientific writing like in-text citations and figure and table cross references.  
- We used manubot, which works with GitHub to automate rendering of formatted document with every edit (PR)
- Other tools that can do these things (e.g. citations and cross-refs) and also combine code and the output of code (e.g. figures).  These include R Markdown (using bookdown package), Jupyter Notebooks (probably??), and a relatively new tool, Quarto.

I'd need to do some research on manuscript writing with Jupyter to see if it's a thing.

What do you think?  I think what you've written mostly follows this outline, but maybe could benefit from a little more introducing the challenges and explicitly talking about what challenges manubot and other tools solves (citations and cross-refs, including code, automated building, etc.)

**Comment by Aariq** [comment]
I'm also happy to revise more during the next phase if you want to just merge this now.

**Comment by robcrystalornelas** [comment]
@emmajhudgins I can make @Aariq's suggested edits if you'd like. Just let me know. I think Brandon and I will start our detailed edits Monday/Tuesday of next week

**Comment by emmajhudgins** [comment]
> @emmajhudgins I can make @Aariq's suggested edits if you'd like. Just let me know. I think Brandon and I will start our detailed edits Monday/Tuesday of next week

That would be great @robcrystalornelas! Happy to take a look at your edits afterwards.

**Review by Aariq — APPROVED** [review]
I'm going to approve for now, but I may want to revise this section some in the next round.

**Comment by robcrystalornelas** [comment]
Sounds good @Aariq. I'll get this merged now, and we can work on it next round too.

**Pull request merged**